templates/themes/app/content/macros/video.html.twig

```diff
@@ -29,6 +29,12 @@
 {% macro poster_slide(content, use_lazy_load, alt_text) %}
     {% import '@ibexadesign/content/macros/slide.html.twig' as slide %}
 
+    {% if not content.fields.title.empty %}
+        {% set video_title = content.fields.title.value.text %}
```

Comment:
We don't use variable video_title in this macro, only in the one below

Reply:
replaced varijable video_title